src/rules/noUnboundMethodRule.ts

```diff
@@ -87,20 +120,46 @@ export class Rule extends Lint.Rules.TypedRule {
         return this.applyWithFunction(
             sourceFile,
             walk,
-            {
-                ignoreStatic: this.ruleArguments.indexOf(OPTION_IGNORE_STATIC) !== -1,
-            },
+            parseArguments(this.ruleArguments),
             program.getTypeChecker(),
         );
     }
 }
 
+function parseArguments(args: any[]): Options {
+    const options: Options = {
+        ignoreStatic: false,
+        whitelist: [],
+    };
+
+    for (const arg of args) {
+        if (typeof arg === "string") {
+            if (arg === OPTION_IGNORE_STATIC) {
+                options.ignoreStatic = true;
+            }
+        } else {
+            if (arg.hasOwnProperty(OPTION_IGNORE_STATIC)) {
+                options.ignoreStatic = arg[OPTION_IGNORE_STATIC];
+            }
+            if (arg.hasOwnProperty(OPTION_WHITELIST)) {
+                options.whitelist = options.whitelist.concat(arg[OPTION_WHITELIST]);
```

Comment:
These lines are failing the [lint build on CircleCI](https://circleci.com/gh/palantir/tslint/11243?utm_campaign=vcs-integration-link&utm_medium=referral&utm_source=github-build-link) because `arg` is unsafely being dealt with. See `banRule.ts` for a good example of how this can be done a little more type-safely.

Reply:
I have changed the parsing logic - hope it works now. For some reason I cannot run `yarn lint` locally (it just hangs...).

Reply:
Huh, weird. Might be a separate issue to file?